includes/class-press-sync.php

```diff
@@ -1055,7 +1086,11 @@ public function parse_sync_settings( $settings = array() ) {
 			'preserve_ids'         => get_option( 'ps_preserve_ids', false ),
 			'fix_terms'            => get_option( 'ps_fix_terms', false ),
 			'ps_content_threshold' => get_option( 'ps_content_threshold', false ),
+			'ps_short_terms'       => get_option( 'ps_short_terms', false ),
+			'ps_page_size'         => get_option( 'ps_page_size', self::PAGE_SIZE ),
```

Comment:
we dont use $this->settings['ps_page_size'] here?

Reply:
disregard my comment.

Reply:
It's used in `class-dashboard.php` when we're getting the page size for the count callback - this allows us to pass it to the Javascript. In the future we'll include this in the localized data going to Javascript.